frontend-site/src/modules/staticData.js

```diff
@@ -1,29 +1,72 @@
 // TODO: decide what stays here and what can be stored as a JSON file for the API to send
 export const utilitiesOffered = Object.freeze([
+  'Hardware',
+  '3-D Printing',
+  '3-D Scanning',
   'Raspberry Pi',
   'Arduino',
+  'LUG API',
+  'Board Games',
+  'Game Systems',
   'Ice Cream',
-  '3-D Printing',
 ]);
 
 export const servicesOffered = Object.freeze([
   // TODO: fill in empty entries
   {
-    name: 'Raspberry Pi',
-    image: '',
-    description: '',
+    name: 'Hardware',
+    image: require("@/assets/Hardware.png"),
+    description: 'From resistors to soldering irons, the LUG is stocked with a wide variety of hardware componants available for student use.',
     url: '',
   },
+  {
+    name: '3D Printing',
```

Comment:
Still have inconsistent use of `3D` versus `3-D` here and in the `3-D Scanning` entry. You also use `3-D` in the utilities listing.